src/mimeparser.rs

```diff
@@ -69,6 +69,10 @@ pub struct MimeMessage {
     // if this flag is set, the parts/text/etc. are just close to the original mime-message;
     // clients should offer a way to view the original message in this case
     pub is_mime_modified: bool,
+
+    // this is non-empty only if the message was actually encrypted.
+    // used to save the decrypted, raw mime structure as needed eg. for late-parsing HTML.
```

Comment:
Why not a real doc comment?
```suggestion
    /// The decrypted, raw mime structure.
    ///
    /// This is non-empty only if the message was actually encrypted.  It is used
    /// for e.g. late-parsing HTML.
```